src/api/v3/feedback/region_list_feedback.py

```diff
@@ -0,0 +1,11 @@
+from django.http import JsonResponse
+
+from api.decorators import feedback_handler
+from cms.models import RegionListFeedback
+
+
+@feedback_handler
+# pylint: disable=unused-argument
+def region_list_feedback(data, region_slug, language_code, comment, emotion, is_technical):
```

Comment:
If we use this endpoint without the`region_slug` and `language_code` parameters, you'll probably need adjustments to `@feedback_handler`...